lib/offline.js

```diff
@@ -149,12 +156,10 @@ module.exports = function(config) {
       try {
         stat = fs.statSync(filepath);
       } catch (ex) {
-        process.stdout.write(chalk.red.bold(filepath + ' doesn\'t exist.'));
         throw ex;
       }
```

Comment:
Without the *process.stdout.write* call, there's no reason to catch the exception that *fs.statSync* throws if the path doesn't exist, so this should simply call fs.statSync and let the exception propagate automatically.

But once you do that, there's little value to factoring out the two checks into *assertExists* at all, since it only has one caller and is only four lines long:

```
function assertExists() {
  var stat = fs.statSync(filepath);
  if (!stat.isFile()) {
    throw new Error(filepath + ' is not a file.');
  }
}
```

So I would unrefactor it and move its logic into *checkImports*. I would then also continue to distinguish between the two possible error states instead of lumping them together into `doesn't exist or is a directory`, since we know which one is the case, and a more-specific error message will be less confusing.